app/src/main/java/org/helfoome/presentation/MainActivity.kt

```diff
@@ -1,12 +1,48 @@
 package org.helfoome.presentation
 
 import android.os.Bundle
+import com.naver.maps.geometry.LatLng
+import com.naver.maps.map.*
+import com.naver.maps.map.overlay.Marker
+import com.naver.maps.map.overlay.OverlayImage
 import org.helfoome.R
 import org.helfoome.databinding.ActivityMainBinding
 import org.helfoome.util.binding.BindingActivity
 
-class MainActivity : BindingActivity<ActivityMainBinding>(R.layout.activity_main) {
+class MainActivity : BindingActivity<ActivityMainBinding>(R.layout.activity_main), OnMapReadyCallback {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        initNaverMap()
+    }
+
+    private fun initNaverMap() {
+        val fm = supportFragmentManager
+        val mapFragment = fm.findFragmentById(R.id.fragment_naver_map) as MapFragment?
+            ?: MapFragment.newInstance().also {
+                fm.beginTransaction().add(R.id.fragment_naver_map, it).commit()
```

Comment:
네이버 맵 프래그먼트라서 적용이 될지는 모르겠지만.. `fragment-ktx` 활용해서, transaction을 보다 간결히 표현가능합니다!
```suggestion
supportFragmentManager.commit {
    add<MapFragment>(R.id.fragment_naver_map)
    setReorderingAllowed(true)
}
```